cryptoki/src/context/slot_token_management.rs

```diff
@@ -7,17 +7,17 @@ use crate::error::{Result, Rv};
 use crate::label_from_str;
 use crate::mechanism::{MechanismInfo, MechanismType};
 use crate::slot::{Slot, SlotInfo, TokenInfo};
-use cryptoki_sys::{CK_MECHANISM_INFO, CK_SLOT_INFO, CK_TOKEN_INFO};
+use cryptoki_sys::{CK_BBOOL, CK_MECHANISM_INFO, CK_SLOT_INFO, CK_TOKEN_INFO};
 use std::convert::TryInto;
 
 // See public docs on stub in parent mod.rs
 #[inline(always)]
-pub(super) fn get_slots_with_token(ctx: &Pkcs11) -> Result<Vec<Slot>> {
+pub(super) fn get_slots(ctx: &Pkcs11, with_token: CK_BBOOL) -> Result<Vec<Slot>> {
```

Comment:
Why use `CK_BBOOL` rather than `bool`?  Just to avoid the conversion from `bool` to `CK_BBOOL`?

Reply:
Basically. It's a value headed into the FFI boundary, and it's not publicly visible, so there's no real reason to introduce a conversion.